File: T5. C# CONDITIONAL STATEMENTS AND LOOPS (Java)/P05_ForeignLanguages.java
import java.util.Scanner;

public class P05_ForeignLanguages {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        String countryInput = in.nextLine();

        if (countryInput.equals("USA") || countryInput.equals("England"))
        {
            System.out.println("English");
        }
        else if(countryInput.equals("Spain") || countryInput.equals("Argentina") || countryInput.equals("Mexico"))
        {
            System.out.println("Spanish");
        }
        else
        {
            System.out.println("unknown");
        }
    }
}

File: T5. C# CONDITIONAL STATEMENTS AND LOOPS/P05. ForeignLanguages.cs
using System;

namespace P05._ForeignLanguages
{
    class Program
    {
        static void Main(string[] args)
        {
            string countryInput = Console.ReadLine();

            if (countryInput.Equals("USA") || countryInput.Equals("England"))
            {
                Console.WriteLine("English");
            }
            else if(countryInput.Equals("Spain") || countryInput.Equals("Argentina") || countryInput.Equals("Mexico"))
            {
                Console.WriteLine("Spanish");
            }
            else
            {
                Console.WriteLine("unknown");
            }
        }
    }
}
